helpers/__tests__/userAgent.js

```diff
@@ -0,0 +1,39 @@
+import { USER_AGENTS } from '../constants'
```

Comment:
Filename inconsistent with the other file in this directory.

Reply:
Need to move constants to its own folder at some point. Later PR probably.